docs/buyers-guide.md

```diff
@@ -0,0 +1,20 @@
+# Buyer's Guide
+
+Buyers have two options:
+1. buy a complete hardware kit from the OpenPolScope project, or
+2. assemble your own kit piece by piece. 
+
+## Buy a kit from the OpenPolScope project
+ 
+- Read about the [OpenPolScope Hardware Kit](https://openpolscope.org/pages/OPS_Hardware.htm).
+- Complete the [OpenPolScope information request form](https://openpolscope.org/pages/Info_Request_Form.htm).
+
+## Buy individual components
+
+| Part                     | Approximate Price | Notes                       |
+|--------------------------|-------------------|-----------------------------|
+| Liquid crystal compensator | $6,000 | Meadowlark optics LVR-42x52mm-VIS-ASSY. Although near-variants are listed in the [Meadowlowlark catalog](https://www.meadowlark.com/product/liquid-crystal-variable-retarder/), this is a custom part with two liquid crystals in a custom housing. [Contact Meadowlark](https://www.meadowlark.com/contact-us/) for a quote.|
+| Liquid crystal control electronics | $2,000 | [Meadowlark optics D5020-20V](https://www.meadowlark.com/product/liquid-crystal-digital-interface-controller/). Choose the high-voltage 20V version. 
+| Liquid crystal adaptor | $25-$500 | A 3D printed part that aligns the liquid crystal compensator in a microscope stand's illumination path. [Contact us](compmicro@czbiohub.org) for printer-ready `.stl` files; uncommon stands might require a custom design.|
```

Comment:
We can update this with a link to @amitabhverma's `.stl` files if they are public.

Reply:
@mattersoflight FYI I pulled the `LVR-42x52mm-VIS-ASSY` part number from a 2022 invoice from Meadowlark.

Reply:
Yes, that is the correct part number.

Reply:
Just want to add that there are 2 assembly sizes, the 42x52mm and 50x60mm with 22mm and 30mm clear aperture diameter respectively. For the majority of cases, sans the Zeiss slider which only accommodates 22mm CA we use the bigger CA size assembly which is better suited for most microscopes.

Reply:
@amitabhverma are 3D printed parts designed to fit 50x60mm assembly? Can you please point us to the onshape collection?

Reply:
Yes @mattersoflight, parts are mainly for 50x60mm assembly except for the Zeiss slider since its limited by the microscope slot. I've shared the invite link with @talonchandler. For some reason even though it shows public on my end the OnShape catalog doesn't list them.

Reply:
Hi @amitabhverma I just following the onshape sharing link, and I was still unable to access the files. 

A possible option: would you be open to hosting them on github directly? Github seems to do [a good job of rendering these files](https://github.com/stephenyeargin/stl-files/blob/main/AT%26T%20Building.stl).

Reply:
@talonchandler I'll look into this. I don't see any issues hosting them here.

Reply:
@talonchandler @mattersoflight 
Here's the link for the uploaded .stl files. Feel free to fork/link https://github.com/amitabhverma/Microscope-LC-adapters

Reply:
Awesome! Thanks for doing this @amitabhverma. 

I've just added a link to the new repo of .stl files.